### PR TITLE
Extend prfile of Mzkizi Johnson

### PR DESCRIPTION
**Pull request by MpaulaL**
I added a bit more information about his motivation in learning R. Issue #9

### CODE REVIEW

File: profiles/learner-profiles.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Often, she sees herself struggling with coding problems. That’s ok, just life,
 
  
 
-**Mzkizi Johnson** is an environmental scientist who is used to working with Python. He’s getting involved in a new project that has a lot of R and he needs to learn the ins and outs of it (best practices, etc.). 
+**Mzkizi Johnson** is an environmental scientist who is used to working with Python. He’s getting involved in a new project, which involves the use of statistic packages of R, so he would like to get an introduction to R and the best practices of it.
```
Comment (darudmann) [inline profiles/learner-profiles.md]: Well done @MpaulaL! One suggestion:
```suggestion
**Mzkizi Johnson** is an environmental scientist who is used to working with Python. He’s getting involved in a new project, which involves the use of statistical packages of R, so he would like to get an introduction to R and the best practices of it. 
```

**Review by darudmann — CHANGES_REQUESTED** [review]
Thank you, @MpaulaL! I've made one minor change, but otherwise all is good.

**Review by vansteph — COMMENTED** [review]
Well done!

**Review by vansteph — APPROVED** [review]

**Review by darudmann — APPROVED** [review]
Yes

**Comment by MpaulaL** [comment]
Closes #9

**Pull request merged**